code/game/objects/structures/crates_lockers/closets/secure/scientist.dm

```diff
@@ -60,6 +61,7 @@
 	new /obj/item/clothing/accessory/medal/science(src)
 	new /obj/item/clothing/glasses/welding/superior(src)
 
+
```

Comment:
Is there any reason theres extra lines here added on `64` and `40`? LGTM otherwise

Reply:
really makes no difference